private-templates-service/client/src/components/NavBar/NavBar.tsx

```diff
@@ -55,7 +55,7 @@ const NavBar = (props: NavBarProps) => {
     return (
       <Banner>
         <MobileBanner>
-          <StyledLogo src={Logo} />
+          <StyledLogo aria-label={STRINGS.LOGO_DESCRIPTION} src={Logo} />
```

Comment:
Please correct me if I’m mistaken but shouldn’t it be **ariaLabel**? aria-label would be correct in CSS

Reply:
ariaLabel gives me an error, I double checked that aria-label sets the actual label in the accessibility tab